src/graphics/particle/MagicFlare.cpp

```diff
@@ -522,7 +522,7 @@ void FlareLine(Vec2f fromPos, Vec2f toPos, Entity * io) {
 			long z = Random::get(0, FLARELINERND);
 			z += FLARELINESTEP;
 			if(!io) {
-				z = long(z * g_sizeRatio.y);
+				z *= long(g_sizeRatio.y);
```

Comment:
Same, I don't think the change of rounding from after to before the multiplication is wanted here.

Reply:
My bad. Removed.